src/README.md

```diff
@@ -4,22 +4,22 @@ This subordinate charm provides a way to authenticate in Openstack for
 a specific domain with a Kerberos ticket. This provides an additional 
 security layer. An external Kerberos server is necessary.
 
-The following documentation is useful to understand better the charm 
-implementation:
-
-* https://www.objectif-libre.com/fr/blog/2018/02/26/kerberos-authentication-for-keystone/ 
-* https://jaosorior.dev/2018/keberos-for-keystone-with-mod_auth_gssapi/
+This kerberos subordinate charm is supported on Ubuntu Bionic (18.04 LTS) with the following versions of 
+Openstack:
```

Comment:
For the sake of future README maintenance, please change this to a more "timeless" statement.  ie. "Queen and later" or something similar.